src/components/List/List.jsx

```diff
@@ -0,0 +1,44 @@
+import React, {useState} from 'react';
+import ListElement from "../ListElement/ListElement";
+
+function List() {
+  const [inputVal, setTodo] = useState("");
+  const [todoList, setTodoList] = useState([]);
+
+  const onButtonClick = (e) => {
```

Comment:
Use more telling variable names. Usually, the code gets minified, changing it to `e` soon enough 😉